src/frontend/test_runner/tests/testdata/batch_dist_agg.yaml

```diff
@@ -0,0 +1,168 @@
+# This file is automatically generated. See `src/frontend/test_runner/README.md` for more information.
+- id: create_tables
+  sql: |
+    /* T: UpstreamHash(row_id); Tk: UpstreamHash(k); S: single distribution */
+    create table T  (k int, v int, o int);
+    create index Tk on T(k) include(k, v, o);
+    create materialized view S as select * from T order by o limit 100;
+- id: agg_on_single
+  before:
+  - create_tables
+  sql: |
+    select max(v) as a1 from S;
+  batch_plan: |
+    BatchSimpleAgg { aggs: [max(max(s.v))] }
+      BatchExchange { order: [], dist: Single }
+        BatchSimpleAgg { aggs: [max(s.v)] }
+          BatchScan { table: s, columns: [s.v], distribution: UpstreamHashShard() }
```

Comment:
we should fix it

Reply:
https://github.com/singularity-data/risingwave/issues/4165